### PR TITLE
Create Docker for Horus

### PR DESCRIPTION
**Pull request by raonis**
First comment this docker.

### CODE REVIEW

File: horus/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM phusion/passenger-ruby24:0.9.20
+MAINTAINER "Raoni Sortica" <raoni@raoni.adm.br>
+
+ENV HOME /root
```
Comment (mateusduboli) [inline horus/Dockerfile]: This seems weird

File: horus/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# enable ngnix and passenger (disabled by default)
+RUN rm -f /etc/service/nginx/down
+
+# config for mapping ngnix > passenger
+RUN rm /etc/nginx/sites-enabled/default
+ADD confs/webapp.conf /etc/nginx/sites-enabled/webapp.conf
+
+
+#Ngnix configs
+ADD confs/secret_key.conf /etc/nginx/main.d/secret_key.conf
```
Comment (mateusduboli) [inline horus/Dockerfile]: I would like more clarification on this pipeline

File: horus/confs/secret_key.conf [hunk 3]
```diff
@@ -0,0 +1 @@
+env SECRET_KEY=123456;
```
Comment (mateusduboli) [inline horus/confs/secret_key.conf]: Shouldn't be exposed in docker? So we can override it in the cli

Comment (igorescobar) [inline horus/confs/secret_key.conf]: NASA style secret.

**Comment by igorescobar** [comment]
@ricardolazaro please, remove my access to 99 repos.

**Comment by igorescobar** [comment]
I drink everyday. Shit can happen.

File: horus/Dockerfile (hunk 1 above)
Comment (igorescobar) [inline horus/Dockerfile]: you are running your app as root. not good.

**Pull request merged**